# Summary table of existing packages
|              | [FASTQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) | seqTools |Shortread|[FastQCR](https://cran.r-project.org/web/packages/fastqcr/index.html)|[fastqp](https://github.com/mdshw5/fastqp)|
| :---         | :---:    |  :---:  | :---:  | :---: | :---: |
|Encoding type | yes  | no    |  yes    | yes| |
|Total read(sequence)|yes|yes|yes|yes||
|sequence length    |yes|yes|yes|yes||
|%GC                      |yes||   |yes||
|per base sequence quality|yes| | |yes|yes|
|per tile sequence quality|yes| | | ||
|per sequence quality|yes| yes | |yes||
|per base sequence content|yes|yes | |yes|yes|
|per  sequence GC content|yes|yes | | yes|yes|
|per base N content|yes|yes | |yes||
|sequence length distribution|yes| | |yes||
|Sequence duplication levels|yes| |yes|yes||
|Overrepresented sequence|yes| | |yes||
|Adapter content|yes| | |yes||
|K-mer content|yes|yes |yes |yes|yes|
|collection time|| yes| |||
|identify low-quality read|yes | |yes | ||

